nucliadb_utils/nucliadb_utils/storages/s3.py

```diff
@@ -514,3 +518,25 @@ async def create_bucket(
                 ]
             },
         )
+
+
+def parse_status_code(error: botocore.exceptions.ClientError) -> int:
```

Comment:
would be nice to have some unit testing for this function, wdyt?

Reply:
Sure, I'll add them